aep/general/0191/aep.md.j2

```diff
@@ -6,7 +6,10 @@ read.
 
 ## Syntax
 
-APIs defined in protocol buffers **must** use `proto3` syntax.
+APIs defined in protocol buffers **must** use `proto3` syntax or one of the
```

Comment:
I think we should also add all fields *should* be marked optional? Unless that's in a different AEP already.

Reply:
LGTM!